code/game/objects/structures/orbital_cannon.dm

```diff
@@ -190,13 +195,14 @@
 			inaccurate_fuel = abs(GLOB.marine_main_ship?.ob_type_fuel_requirements[4] - tray.fuel_amt)
 
 	var/turf/target = locate(T.x + inaccurate_fuel * pick(-1,1),T.y + inaccurate_fuel * pick(-1,1),T.z)
-	for(var/i in hearers(WARHEAD_FALLING_SOUND_RANGE,target))
-		var/mob/M = i
-		M.playsound_local(target, 'sound/effects/OB_incoming.ogg', falloff = 2)
+
+	playsound(target, 'sound/effects/OB_warning_announce.ogg', 100, FALSE, 20, 4) //the ground
```

Comment:
Should everyone on the ground be hearing this?

Reply:
something to think about but can do if #[5315](https://github.com/tgstation/TerraGov-Marine-Corps/pull/5315) gets merged

Reply:
and done